easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.3.1-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,169 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.3.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    # required for installing dm-tree
+    ('Bazel', '5.1.1'),
+    ('CMake', '3.23.1'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('SciPy-bundle', '2022.05'),
+    ('PyYAML', '6.0'),
+    ('TensorFlow', '2.11.0', versionsuffix),
+    ('Biopython', '1.79'),
+    ('HH-suite', '3.3.0'),
+    ('HMMER', '3.3.2'),
+    ('Kalign', '3.3.5'),
+    ('jax', '0.3.14', versionsuffix),  # also provides absl-py
```

Comment:
As far as I can tell, there is nothing on EasyBuild side locking us to v0.3.14 for `foss/2022a`. Can we stick to the requirements set by alphafold v2.3.1 and depend on jax v0.3.25? See https://github.com/deepmind/alphafold/blob/a3941673e90b8d1d75c60b16a4b3707ebf7ba527/requirements.txt#L8

I made a PR for jax v0.3.25 in #17700